Sources/AppCoinsSDK/Models/Raws/CurrencyListRaw.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  File.swift
+//  
+//
+//  Created by Graciano Caldeira on 16/08/2024.
+//
+
+import Foundation
+
+internal struct CurrencyListRaw: Codable {
```

Comment:
No need for separate files, add this to the CurrencyRaw file

Reply:
Done